config/locales/en.yml

```diff
@@ -387,6 +389,7 @@ en:
       SP:
         encryption: https://www.docs.verify.service.gov.uk/rotating-your-keys-and-certificates/service-provider-encryption/#rotate-your-service-provider-encryption-key-and-certificate
         signing: https://www.docs.verify.service.gov.uk/rotating-your-keys-and-certificates/service-provider-signing/#rotate-your-service-provider-signing-key-and-certificate
+      instructions: https://www.docs.verify.service.gov.uk/rotating-your-keys-and-certificates/#rotating-your-keys-and-certificates
```

Comment:
this is the same as two lines below, `rotation_documentation`, so maybe used that instead?